lib/getConfig.js

```diff
@@ -7,7 +7,7 @@ const getConfig = (altPath) => {
   const pathString = altPath || path.join(cwd, '.sgcrc');
   let configObject = json.readToObjSync(pathString);
 
-  if (!configObject) {
+  if (configObject) {
```

Comment:
This should be `!configObject`. If `altPath` or `cwd/.sgcrc` does not exist it should read the `package.json`